ftw/simplelayout/profiles/mapblock/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<metadata>
+  <version>1000</version>
+  <dependencies>
+    <dependency>profile-ftw.simplelayout:default</dependency>
```

Comment:
I think this profile is no longer necessary, since you have moved it to the `mapblock` package, right?